java-sdk-logging/pom.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>java-sdk-logging-parent</artifactId>
+  <version>0.1.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+  <parent>
+    <groupId>com.google.api</groupId>
+    <artifactId>gapic-generator-java-pom-parent</artifactId>
+    <version>2.55.1</version><!-- {x-version-update:gapic-generator-java:current} -->
+    <relativePath>../gapic-generator-java-pom-parent</relativePath>
+  </parent>
+  <modules>
+    <module>log4j2-extension</module>
+    <module>logback-extension</module>
+  </modules>
+
+  <properties>
+    <mockito.version>4.11.0</mockito.version>
```

Comment:
We already have a mockito version in Gax, maybe we can extract that to a property and reuse it here? It could be a follow up PR as well.

Reply:
Done